src/remoted/manager.c

```diff
@@ -209,40 +234,159 @@ void save_controlmsg(const keyentry * key, char *r_msg, size_t msg_length)
             /* Unlock mutex */
             w_mutex_unlock(&lastmsg_mutex);
 
-            /* Write uname to the file */
+            /* Parsing uname */
```

Comment:
This code block can be moved to a parser function. And it will be useful when adding the cluster logic.

Reply:
DONE